tests/Cargo.toml

```diff
@@ -22,11 +22,16 @@ name = "wgpu-compile-test"
 path = "compile_tests/root.rs"
 harness = true
 
+[[test]]
+name = "wgpu-validation-test"
+path = "validation_tests/root.rs"
+harness = true
+
 [features]
 webgl = ["wgpu/webgl"]
 
 [dependencies]
-wgpu.workspace = true
+wgpu = { workspace = true, features = ["noop"] }
```

Comment:
This is a non-obvious one, we should move the feature here to the workspace - I've unified all test uses to be straight workspace deps, and it's not a real harm for tests

Reply:
That seems like it would be bad if the workspace ever gains any non-test packages that depend on `wgpu`. (Arguably, `wgpu-info` is one already, though it would be reasonable for it to include noop.)

Reply:
Potentially - I made that change in part because we kept forgetting features between test/wgpu-info (which need to be in sync) and other areas.

I think it's fine for now though.

Reply:
Done.